dag/main.yml

```diff
@@ -366,6 +366,16 @@ steps:
   - backport://backport/owid/latest/dataset_235_world_happiness_report__2022
   data://grapher/happiness/2023-03-20/happiness:
   - data://garden/happiness/2023-03-20/happiness
+  # Ethnic Power Relations Dataset
+  data://meadow/eth/2023-03-15/growup:
+  - snapshot://eth/2023-03-15/growup.csv
+  data://garden/eth/2023-03-15/ethnic_power_relations:
+  - data://meadow/eth/2023-03-15/growup
+  - data://garden/demography/2023-03-31/population
+  - data://garden/reference
```

Comment:
Since you are using `geo.add_region_aggregates`, you also need to include the dependence on income groups:
`- data://garden/wb/2021-07-01/wb_income`